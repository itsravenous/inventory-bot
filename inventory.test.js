const inventoryBot = require("./functions/inventory.js");
it("returns inventory for user", async () => {
  const inventory = await inventoryBot.getInventory("Bilbo");
  expect(inventoryBot.inventoryToText(inventory)).toMatchSnapshot();
});

it("returns inventory for another user", async () => {
  const inventory = await inventoryBot.getInventory("Frodo");
  expect(inventoryBot.inventoryToText(inventory)).toMatchSnapshot();
});
