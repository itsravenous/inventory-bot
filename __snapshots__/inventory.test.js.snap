// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns inventory for another user 1`] = `
"*Body:*
Ring of power

*Small pouch:*
Lembas bread

"
`;

exports[`returns inventory for user 1`] = `
"*Body:*
Sting (d15)
Pipe

*Sack:*
Seed cake

"
`;
